scipy/optimize/tests/test_chandrupatla.py

```diff
@@ -482,7 +483,7 @@ def test_bracket_order(self, xp):
         loc = xp.linspace(-1, 1, 6)[:, xp.newaxis]
         brackets = xp.asarray(list(permutations([-5, 0, 5]))).T
         res = _chandrupatla_minimize(self.f, *brackets, args=(loc,))
-        assert xp.all(xp.isclose(res.x, loc) | (res.fun == self.f(loc, loc)))
+        assert xp.all(xpx.isclose(res.x, loc) | (res.fun == self.f(loc, loc)))
```

Comment:
This accidentally worked before because the test is skipping array-api-strict for unrelated reasons